compose.yml

```diff
@@ -11,11 +11,7 @@ services:
     ports:
       - "8002:80"
 
```

Comment:
why are you deleting the block below

Reply:
@s7rosine what is your take on this

Reply:
Good to go

Reply:
The Project is sweet

Reply:
hummm I dont  think so

Reply:
@devopseasylearning, i think this was discussed during previous meeting

Reply:
i like this

Reply:
block of code

Reply:
Agreed

Reply:
ok. my comment is already there

Reply:
Good job!

Reply:
finally!
 i was able to comment....

Reply:
Ok to merge